vb6/VisualBasic6.g4

```diff
@@ -35,93 +36,96 @@
 * Change log:
 *
 * v1.4
-*	- erase statement fix
-*	- explicit token definition
+*  - erase statement fix
+*  - explicit token definition
 *
 * v1.3
-*	- call statement precedence
+*  - call statement precedence
 *
 * v1.2
-*	- refined call statements
+*  - refined call statements
 *
 * v1.1 
-*	- precedence of operators and of ELSE in select statements
-*	- optimized member calls
+*  - precedence of operators and of ELSE in select statements
+*  - optimized member calls
 *
 * v1.0 Initial revision
 */
 
 grammar VisualBasic6;
 
+options
+{
+   language = Java;
+}
+
+
 // module ----------------------------------
-startRule
-   : module EOF
-   ;
-
-module
-   : (moduleHeader)? moduleConfig? moduleAttributes? moduleOptions? moduleBody?
-   ;
-
-moduleHeader
-   : VERSION DOUBLELITERAL CLASS
-   ;
-
-moduleConfig
-   : BEGIN moduleConfigElement + END
-   ;
-
-moduleConfigElement
-   : ambiguousIdentifier EQ literal
-   ;
-
-moduleAttributes
-   : (attributeStmt) +
-   ;
-
-moduleOptions
-   : (moduleOption) +
-   ;
-
-moduleOption
-   : OPTION_BASE INTEGERLITERAL # optionBaseStmt
-   | OPTION_COMPARE (BINARY | TEXT) # optionCompareStmt
-   | OPTION_EXPLICIT # optionExplicitStmt
-   | OPTION_PRIVATE_MODULE # optionPrivateModuleStmt
-   ;
-
-moduleBody
-   : moduleBodyElement (moduleBodyElement)*
-   ;
-
-moduleBodyElement
-   : moduleBlock
+
+startRule : module EOF;
+
+module : 
+   WS? NEWLINE*
```

Comment:
In Visual Basic 6.0 whitespace is relevant for semantics (cf. #487 ). 

```
WITH Customer
    SomeProcedure .Name
END WITH

WITH Customer
    SomeObject.Name
END WITH
```

With WS in the grammar SomeProcedure .Name is parsed as ICS_B_ProcedureCall and SomeObject.Name as ICS_B_MemberProcedureCall. Without WS both calls are parsed as ICS_B_MemberProcedureCall. Distinguishing these both cases is relevant for semantic analysis of the AST, i. e. whether to look up procedures or variables.